test/groovy/MTABuildTest.groovy

```diff
@@ -144,9 +122,9 @@ public class MtaBuildTest extends BasePiperTest {
     void noIdInMtaTest() {
 
         thrown.expect(AbortException)
-        thrown.expectMessage("Property 'ID' not found in mta.yaml file at: '")
+        thrown.expectMessage("Property 'ID' not found in mta.yaml file.")
 
-        mtaYaml.text = noIdMtaYaml()
+        jryr.registerYaml('mta.yaml', { noIdMtaYaml() })
```

Comment:
No need for a closure anymore

Reply:
done

Reply:
This is not yet resolved, correct? @marcusholl

Reply:
ups, mistake, not it is resolved. Thanks for the hint.